web_tool_script_1.R

```diff
@@ -95,6 +95,11 @@ if (new_data == TRUE) {
 ####################
 #### PORTFOLIOS ####
 ####################
+abort_if_file_doesnt_exist(
+  here::here(
+    "working_dir", "20_Raw_Inputs", glue::glue("{portfolio_name_ref_all}.csv")
+  )
+)
 portfolio_raw <- get_input_files(portfolio_name_ref_all)
```

Comment:
I do think it might be better if `abort_if_file_doesnt_exist()` were integrated into `get_input_files()` rather than being run separately... but that opens a can of worms because `get_input_files()` is currently defined in `0_web_functions.R`

Reply:
I agree that in the long run `get_input_files()` could itself assert inputs. But because we have so few tests, right now I prefer to add changes in a way that minimize their effects on the system.